tests/contrib/langchain/test_langchain_community.py

```diff
@@ -1267,7 +1267,46 @@ def test_lcel_chain_non_dict_input(langchain_core):
     sequence.invoke(1)
 
 
-@pytest.mark.snapshot
+@pytest.mark.snapshot(ignores=["metrics.langchain.tokens.total_cost"])
```

Comment:
For the sake of our flaky langchain community tests, let's add a `@flaky` marker for the new tests while we figure out how to resolve this 😄

Reply:
so true 🥲 will do!